site/_data/releases.yaml

```diff
@@ -1,7 +1,7 @@
 stable:
-  name: Darga-3
-  tag: darga-3
-  filename: darga-3
+  name: Darga-4
+  tag: darga-4.1
+  filename: darga-4.1
```

Comment:
We can do the same with prereleases here, and I'll add a section in the download page conditional on the prerelease info.